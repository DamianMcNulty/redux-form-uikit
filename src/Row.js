
const Row = () => <div />

export default Row
