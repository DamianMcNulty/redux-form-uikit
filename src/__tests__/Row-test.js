import renderer from 'react-test-renderer'

import Row from '../Row'

describe('Row', () => {
  it('renders correctly', () => {
    expect(renderer.create(
      <Row />
    ).toJSON()).toMatchSnapshot()
  })
})
