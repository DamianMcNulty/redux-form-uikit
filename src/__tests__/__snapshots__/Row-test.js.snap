exports[`Row renders correctly 1`] = `<div />`;
